core/src/main/scala/kafka/raft/RaftManager.scala

```diff
@@ -313,4 +314,8 @@ class KafkaRaftManager[T](
   override def leaderAndEpoch: LeaderAndEpoch = {
     client.leaderAndEpoch
   }
+
+  override def idToNode(id: Int, listener: String): Option[Node] = {
+    client.idToNode(id, listener).toScala
+  }
```

Comment:
I prefer if we call this something like `voterNode` as it makes it clear that this only resolves to a `Option[Node]` if the id is a voter/controller.

Reply:
ack